reconcile/openshift_serviceaccount_tokens.py

```diff
@@ -35,6 +36,31 @@ def construct_sa_token_oc_resource(name: str, sa_token: str) -> OR:
     )
 
 
+def service_account_token_request(name: str) -> OR:
+    """Create a service account token secret for a given service account."""
+    body = {
+        "apiVersion": "v1",
+        "kind": "Secret",
+        "type": "kubernetes.io/service-account-token",
+        "metadata": {
+            # service-account-name-<random-number>
+            "name": f"{name}-token-{random.randrange(99999):05d}",
```

Comment:
Should we use a fixed name suffix to have a one-to-one mapping between legacy service account token and account, which means don't use any previously auto generated ones. So whenever we expect a service account token, we know the desired name, don't need to list current state then select.

Reply:
I've created a ticket https://issues.redhat.com/browse/APPSRE-10565 for that to discuss.